src/api/app/views/webui/projects/maintained_projects/index.html.haml

```diff
@@ -0,0 +1,37 @@
+:ruby
+  @pagetitle = "Projects Maintained by #{@project}"
+  update_policy = policy(@project).update?
+
+.card
+  .card-body
+    - if update_policy
+      %ul.list-inline
+        %li.list-inline-item.mr-3
+          = link_to('#', data: { toggle: 'modal', target: '#new-maintenance-project-modal' }) do
+            %i.fas.fa-plus-circle.text-primary
+            Add Project
```

Comment:
Just realized after commenting on the modal, this could be improved. I would suggest `Add Project to Maintain`

Reply:
I think "Enable/Disable" maintenance is better. What do you think @dmarcoux ?

Reply:
Enable/Disable Maintenance for a Project?

Reply:
yes @dmarcoux, like https://github.com/openSUSE/open-build-service/pull/7379/files#diff-24a73bdcd40dc8e0831830b49a4ed8d4R27

Reply:
Sounds good